cli/cmd/contracts.go

```diff
@@ -86,11 +87,19 @@ var (
 			code, err := conn.Runtime(npa.ParaTime).Contracts.Code(ctx, client.RoundLatest, contracts.CodeID(codeID))
 			cobra.CheckErr(err)
 
+			// Fetch WASM contract code, if supported.
```

Comment:
I would make this a separate operation instead of dumping the code on every `show-code`.

Reply:
Good idea. I added `contracts dump-code`.